env.js

```diff
@@ -90,7 +91,13 @@ function setVersion() {
 }
 
 function setStorage() {
-  env.storageURI = readENV('STORAGE_URI') || readENV('MONGO_CONNECTION') || readENV('MONGO') || readENV('MONGOLAB_URI') || readENV('MONGODB_URI');
+  var appEnv = cfenv.getAppEnv();
+  var svcs = appEnv.services;
+  var uri = null;
+  if (svcs.hasOwnProperty('mlab')) {
+    uri = svcs.mlab[0].credentials.uri;
+  }
+  env.storageURI = readENV('STORAGE_URI') || readENV('MONGO_CONNECTION') || readENV('MONGO') || readENV('MONGOLAB_URI') || readENV('MONGODB_URI') || uri;
```

Comment:
Please refactor so that people who don't have `cf` or `svcs.hasOwnProperty('mlab')` are not affected by this feature. 
You're creating an `uri`, which is only used for a very small percentage of the users. Please use `if` statement to only use that when people use Pivotal Web Services Cloud Support.